src/index.html

```diff
@@ -1,24 +1,35 @@
 <!--
-  ~ Copyright © Live-Poll 2020. All rights reserved
+  ~ Copyright © Live-Poll 2020-2021. All rights reserved
   -->
 
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="utf-8">
     <title>LivePoll - Create live polls for your presentation</title>
     <base href="/">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=0.9">
```

Comment:
why?

Reply:
I was experimenting to make the UI look better on mobile devices. 0.9 as initial scale factor was the best compromise between readability and component squeezing ;)

Reply:
I just added a default factor of 1 and only for mobile devices a factor of 0.9. I think this is better